### DISCUSSION TITLE
Do I have to assume this integration is now obsolete

### DISCUSSION THREAD

**Discussion by chesterflaps** [alandtse/alexa_media_player#2623] [Q&A]
A couple of weeks ago, the integration stopped working. All of my devices say unavailable and the automations I have setup to announce from the ECHO devices, does not work anymore.

I have tried reconfiguring, reloading, disabling and re-enabling, deleting and reinstalling the integration but still, all of the devices are unavailable.

I even tried reverting to an older version of the integration and still, it did not work.

Is anyone else having the same issues?

**Comment by slackstarr** [comment]
I am facing the same exact issues you are describing. I deleted the whole integration several times, and now it's not even showing up in the devices & services section in HA anymore. It looks like that this whole thing is unfortunately broken.

**Comment by Avpman2** [comment]
In the repo it says _Please note this mimics the Alexa app but Amazon may cut off access at anytime._ Maybe they've done just that. Has anyone had any luck?

**Comment by Avpman2** [comment]
In the repo it says _Please note this mimics the Alexa app but Amazon may cut off access at anytime._ Maybe they've done just that. Has anyone had any luck?

**Comment by danielbrunt57** [comment]
My installation is working fine...

**Comment by danielbrunt57** [comment]
Are you running...

![image](https://github.com/user-attachments/assets/5136a3f6-415a-435f-812c-1e20c2cc5d95)

**Comment by alcidesteixeira** [comment]
I've just tried with v5.4 and actually v5.5 that just updated and no luck. Also having the same `Unavailable` message on my echo devices.
Just in case there's something to do with the region, I'm using amazon.es

**Reply by alcidesteixeira** [reply]
Just an update. For my issue, I just had to change `amazon.es` with `amazon.com` and my devices came back available.
